### DISCUSSION TITLE
Enhanced Dropdown - Add Search

### DISCUSSION THREAD

**Discussion by eblanchette** [plausible/analytics#1134] [Anything Else]
The website domain dropdown is ineffective when dealing with hundreds of domains.  In our case, it takes a while to open.

![plausible](https://user-images.githubusercontent.com/704551/122427717-57fd6880-cf5f-11eb-9cd3-57e09397d8fa.jpg)

Thanks,

Eric

**Comment by metmarkosaric** [comment]
thanks for the feedback @eblanchette! we'll consider if we can shorten the list and then perhaps allow you to search rather than keep scrolling

**Comment by ukutaht** [comment]
Yeah this makes sense in case you're dealing with many sites
